test/test-patientsController.js

```diff
@@ -896,7 +896,7 @@ describe('patientsController', () => {
       .set('Cookie', `token=${cookie.getJwtTokenForUser(psy.dossierNumber, 'randomXSRFToken')}`)
       .set('xsrf-token', 'randomXSRFToken')
         .then(async (res) => {
-          res.body.success.should.equal(false);
+          res.status.should.equal(400);
```

Comment:
Maybe 404 Not found? (No difference between invalid and not linked to the user)

Reply:
same => invalid uuid